src/TBuffer.cpp

```diff
@@ -706,6 +706,14 @@ void TBuffer::translateToPlainText(std::string& incoming, const bool isFromServe
         }
 
 COMMIT_LINE:
+        // some rudimentary handling of backspace:
+        if (ch == '\b') {
+            ++localBufferPosition;
```

Comment:
This isn't related to MXP and could mess with how Mudlet works for other games. Could you submit this as a separate PR, so we can test it on its own?

Reply:
Ok, yes this also and indeed of minor importance in this contect. I'll single this out. I'm new to GitHub, so I might need some time seeing how to do it (probably I need to make a different branch).

For the remaining comments, they seem legit too.. I'll work this out and make another Pull request.. I think that's the right procedure (again, I'm new to githubt).

Reply:
> I'm new to GitHub, so I might need some time seeing how to do it (probably I need to make a different branch).

Well, welcome to the party! 😄 Yes, you can make lots of branches, they are easy to manage. Do one for each PR. Make many small PRs, they are easier to review. Also, this may help as well: https://github.com/Mudlet/Mudlet/blob/development/.github/CONTRIBUTING.md